src/components/Playlist/VideoPlaylist.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo, useState } from 'react';
+import { selectVideoPlaylist } from '@100mslive/hms-video-store';
+import { useHMSTheme } from '../../hooks/HMSThemeProvider';
+import { hmsUiClassParserGenerator } from '../../utils/classes';
+import { CloseIcon, PlaylistIcon } from '../Icons';
+import { Text } from '../Text';
+import { useHMSActions, useHMSStore } from '../../hooks/HMSRoomProvider';
+import { Button } from '../Button';
+import { ContextMenu, ContextMenuItem } from '../ContextMenu';
+import { PlaylistItem } from './PlaylistItem';
+
+export interface VideoPlaylistClasses {
+  root?: string;
+  header?: string;
+  body?: string;
+  footer?: string;
+  collapse?: string;
+  controls?: string;
+  icon?: string;
+  sliderContainer?: string;
+}
+
+export interface VideoPlaylistItemClasses {
+  listItem?: string;
+  titleContainer?: string;
+  truncate?: string;
+  selection?: string;
+}
+
+export interface VideoPlaylistProps {
+  classes?: VideoPlaylistClasses;
+  onClose?: () => void;
```

Comment:
Is this used?

Reply:
updated. removed for now. will add it when necessary.